ui-components/src/forms/HouseholdMemberForm.tsx

```diff
@@ -1,34 +1,30 @@
-import React, { useContext } from "react"
-import { NavigationContext } from "../config/NavigationContext"
-import { HouseholdMemberUpdate } from "@bloom-housing/backend-core/types"
+import React from "react"
 import { t } from "../helpers/translator"
 import { Icon, IconFillColors } from "../icons/Icon"
 import { ViewItem } from "../blocks/ViewItem"
 
-const HouseholdMemberForm = (props: {
-  member: HouseholdMemberUpdate
-  type: string
+export interface HouseholdMemberFormProps {
+  editMember?: (memberId: number | undefined) => void
   editMode?: boolean
-}) => {
-  const { member, type } = props
-  const { router } = useContext(NavigationContext)
-  const editMode = props.editMode !== false // undefined should default to true
+  memberFirstName: string
+  memberId?: number
+  memberLastName: string
+  subtitle: string
+}
+
+const HouseholdMemberForm = (props: HouseholdMemberFormProps) => {
+  const editMode = props.editMode !== false && props.editMember // undefined should default to true
 
-  const editMember = () => {
-    if (member.orderId != undefined && member.orderId >= 0) {
-      void router.push({
-        pathname: "/applications/household/member",
-        query: { memberId: member.orderId },
-      })
-    } else {
-      void router.push("/applications/contact/name")
-    }
-  }
   return (
-    <ViewItem helper={type} className="pb-4 border-b text-left">
-      {member.firstName} {member.lastName}
+    <ViewItem helper={props.subtitle} className="pb-4 border-b text-left">
+      {props.memberFirstName} {props.memberLastName}
       {editMode ? (
-        <a id="edit-member" className="edit-link" href="#" onClick={editMember}>
+        <a
```

Comment:
While you're in here…this should probably be changed to `button` and not be an `a` since it doesn't actually go to a real href.

Reply:
updated!